ironfish/src/wallet/wallet.ts

```diff
@@ -888,6 +887,36 @@ export class Accounts {
     }
   }
 
+  async getTransactionStatus(
```

Comment:
should the transaction status be a enum type instead of free form string?

Reply:
I think that would be helpful -- I think we need to standardize transaction statuses across the CLI and our documentation